### PR TITLE
chore: use standard ES6 imports

### PR DESCRIPTION
**Pull request by NathanWalker**
Nice to use standard imports and a little organization in the view model.

### CODE REVIEW

File: main-view-model.ts [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
-    get message(): string {
-        return this._message;
-    }
-    set message(value: string) {
-        if (this._message !== value) {
-            this._message = value;
-            this.notifyPropertyChange("message", value)
-        }
-    }
+  constructor() {
```
Comment (NathanWalker) [inline main-view-model.ts]: Certainly debatable but I find having a class:
1. properties:
  - public
  - private
2. constructor
3. setters/getters
4. public methods
5. private methods

File: main-page.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { HelloWorldModel } from "./main-view-model";
 
 // Event handler for Page "navigatingTo" event attached in main-page.xml
 export function navigatingTo(args: EventData) {
-    // Get the event sender
-    var page = <Page>args.object;
-    page.bindingContext = new HelloWorldModel();
+  // Get the event sender
+  var page = <Page>args.object;
```
Comment (hshristov) [inline main-page.ts]: Even better if `var` is changed to `let`

**Comment by hshristov** [comment]
LGTM

File: app.ts [hunk 3]
```diff
@@ -1,2 +1,2 @@
-﻿import application = require("application");
-application.start({ moduleName: "main-page" });
+﻿import * as app from 'application';
+app.start({ moduleName: "main-page" });
```
Comment (tjvantoll) [inline app.ts]: Single quotes and double quotes in the same file 😮

File: main-view-model.ts [hunk 4]
```diff
@@ -1,38 +1,39 @@
...
-        this._counter = 42;
-        this.updateMessage();
+  get message(): string {
+    return this._message;
+  }
+  
+  set message(value: string) {
+    if (this._message !== value) {
+      this._message = value;
+      this.notifyPropertyChange("message", value)
```
Comment (tjvantoll) [inline main-view-model.ts]: Numerous double quotes in this file. I’m cool with single quotes or double quotes, but before we merge this in can you verify the whole app is consistent on way or the other?

**Comment by NathanWalker** [comment]
@hshristov @tjvantoll got both your comments addressed now, good stuff, thanks!
Feel free to merge when either of you can 👍

**Pull request merged**